tests/test-cases/Group23-VIC-Machine-Service/23-03-VCH-Create.robot

```diff
@@ -54,10 +61,36 @@ Inspect VCH Config ${name}
 Get VCH ${name}
     Get Path Under Target    vch
     ${id}=    Run    echo '${OUTPUT}' | jq -r '.vchs[] | select(.name=="${name}").id'
+    Set Test Variable    ${id}
 
     Get Path Under Target    vch/${id}
 
 
+Pull Busy Box
+    [Arguments]    ${docker_params}    ${expected_rc}    ${expected_output}
+
+    ${rc}  ${output}=  Run And Return Rc And Output  docker -H ${docker_params} pull busybox
+
+    Should Be Equal As Integers    ${rc}    ${expected_rc}
+    Should Contain    ${output}    ${expected_output}
+
+
+Get Docker Host Params
+    [Arguments]    ${vch_name}
+    Run Keyword And Continue On Failure    Wait Until Keyword Succeeds    30x    10s    Get Docker Params API    ${vch_name}
+    Run Keyword If    "${docker_host}" == "null"    Run VIC Machine Inspect Command   ${vch_name}
```

Comment:
do we expect the `Get Docker Params API` to fail? If it's fault injection, ok... but I don't see it here.
Get it from both sources and check they're the same... that's a viable test.

Reply:
This was an intention decision by the author of the test, as discussed here: https://github.com/vmware/vic/pull/7727#discussion_r181500754

Reply:
Added on master: #8089